packages/editor/src/components/visual-editor/index.js

```diff
@@ -363,10 +366,15 @@ function VisualEditor( {
 					// Some themes will have `min-height: 100vh` for the root container,
 					// which isn't a requirement in auto resize mode.
 					enableResizing ? 'min-height:0!important;' : ''
-				}}`,
+				}}${
+					// Vertically expands the top-level post content block when zoomed out on an empty post.
+					isEditedPostEmpty && isZoomedOut
+						? '.entry-content {min-height: 33vh;}'
```

Comment:
I'm so sorry, but I don't like this. This is pseudo-expanding. :) Either it will push the footer beyond the fold, or it will never reach the bottom. The previous implementation expanded the content exactly to the bottom and pushed the footer down. Let's restore that

Reply:
> Either it will push the footer beyond the fold,

In the interest of this being a functional feature providing a larger drop zone, that seems like the right thing to do. If the feature is meant purely as a visual effect I see your point. Though, I think that could be considered a shortcoming of the prior version.

> or it will never reach the bottom.

The footer always reaches the bottom of the canvas in this PR, but I guess you are talking about the canvas reaching the bottom of the viewport. Keeping in mind this is only relevant when there’s not enough content to create overflow in the zoomed out view—it’s not like it was a purely beneficial thing. It creates a visual shift spoiling an otherwise fluid zoom transition and that visual shift seems at odds with the idea of a 1:1 view. Of course, given an empty post this branch also creates a shift, but the previous implementation did regardless.

> I'm so sorry

Please don’t be 🤗. Thank you for the feedback. For me, this is an improvement beyond just a restoration. Not that there isn’t room to improve further.

Reply:
I guess a middle ground could be to restore the full canvas and content expansion but only on empty posts.